### ISSUE TITLE
When enabling private clusters for AKS, the linked AKS cluster provisioning documentations redirects to outdated documentation

### ISSUE THREAD

**Issue by markusewalker** [rancher/dashboard#6209]
**Setup** 
- Rancher version: `v2.6-head` https://github.com/rancher/rancher/commit/772b2f0232764ec1f0f9f90d938a69cce647dcdb
- Browser type & version: Chrome

**Describe the bug**

When creating a private AKS cluster in Rancher, you will receive the following warning message after clicking `Enable Private Cluster`:

![image](https://user-images.githubusercontent.com/52591819/174868887-8bef910f-3099-4e6f-a09e-17c8c76834d2.png)

When I click the hyperlink mentioned in the above mentioned screen capture, I am incorrectly taken to the `v2.5` version of the documentation when I should be taken to the latest `v2.6` version of the documentation.

**To Reproduce**

1. Setup Rancher and navigate to the UI in a browser.
2. In Cluster Management, click `Create` and select `Azure AKS`.
3. Follow through the prompts and when you get to the `Networking Options` section, click `Enable Private Cluster`.
4. Once you see the warning message, click the hyperlink. You'll be incorrectly taken to the `v2.5` version of the documentation instead of the latest `v2.6` version.

**Result**
You're taken to outdated `v2.5` documentation.

**Expected Result**

I should be taken to the latest `v2.6` documentation.

**Comment by nwmac** [comment]
PR with fix: https://github.com/rancher/ui/pull/4862

**Comment by markusewalker** [comment]
Verified that this is addressed in `v2.6-head` https://github.com/rancher/rancher/commit/da21dbb6f3ac5e0ede98f469d95ed2b0d323ba5c.

**ENVIRONMENT DETAILS**
- Rancher Install: Helm
- Rancher version: `v2.6-head` https://github.com/rancher/rancher/commit/da21dbb6f3ac5e0ede98f469d95ed2b0d323ba5c
- Browser: Chrome

**TEST RESULT**
PASS

**VERIFICATION STEPS**
1. Setup Rancher and navigated to the UI in a browser.
2. In Cluster Management, clicked `Create` and selected `Azure AKS`.
3. Followed through the prompts and when I got to the `Networking Options` section, clicked `Enable Private Cluster`.
4. Clicked the resulting hyperlink and was correctly redirected to the latest documentation:
![Jul-20-2022 08-46-36](https://user-images.githubusercontent.com/52591819/180026246-1396e6cc-4fa6-4f69-acb5-853e8c0c8546.gif)
